test/ret/node_stat_test.exs

```diff
@@ -0,0 +1,100 @@
+defmodule Ret.NodeStatTest do
+  use Ret.DataCase
+  alias Ret.{NodeStat, Repo}
+
+  describe "NodeStat.max_ccu_for_time_range() tests" do
+    @tag marked: true
+    test "Get max correctly for today" do
+      seed_today_node_stats([0, 10, 5])
+
+      %{today: today} = get_times()
+
+      assert 10 == NodeStat.max_ccu_for_time_range(today.start_time, today.end_time)
+    end
+
+    @tag marked: true
+    test "Return 0 for no NodeStats collected for today" do
+      # Do not seed database
+      %{today: today} = get_times()
+      assert 0 == NodeStat.max_ccu_for_time_range(today.start_time, today.end_time)
+    end
+
+    @tag marked: true
+    test "Do not return outside of time boundary" do
+      seed_today_node_stats([0, 10, 5])
+      seed_tomorrow_and_yesterday_node_stats(20)
```

Comment:
nit: I do think it's nice to keep test code readable like this, but there is also a philosophy that you should minimize the number of helper functions you use, so that you can read the actual behavior of a test without having to jump around to read the behavior of a bunch of helper functions as well. As a middle-ground, I might have used a single generalized helper, like:
```elixir
%{today: today, tomorrow: tomorrow, yesterday: yesterday} = get_times()

seed_node_stats([
  [yesterday, [20]],
  [today, [0, 10, 5],
  [tomorrow, [20]]
])
```